ugvc/mrd/featuremap_utils.py

```diff
@@ -254,6 +272,24 @@ def edit_vcf_header(self, header: pysam.VariantHeader) -> pysam.VariantHeader:
             f'Number=1,Type=String,Description="{self.motif_length_to_annotate}'
             'bases in the reference after variant">'
         )
+        header.add_line(
+            f"##INFO=<ID={self.NEXT_1}," f'Number=1,Type=String,Description="1 base in the reference after variant">'
+        )
+        header.add_line(
+            f"##INFO=<ID={self.NEXT_2}," f'Number=1,Type=String,Description="2 bases in the reference after variant">'
```

Comment:
NEXT_2 is the second base right? here it say '2 bases'

Reply:
corrected